src/components/core/PivotTable.tsx

```diff
@@ -153,14 +246,46 @@ export class PivotTableInner extends
         }
     }
 
+    /**
+     * getCellClass returns class for drillable cells. (maybe format in the future as well)
+     */
+    public getCellClass = (classList: string) => (cellClassParams: CellClassParams): string => {
+        const { drillableItems } = this.props;
+        // return none if no drillableItems are specified
+        if (drillableItems.length === 0) {
+            return null;
+        }
+
+        const afm: AFM.IAfm = this.props.dataSource.getAfm();
```

Comment:
`datasource.getAfm()` if the spred above is used

Reply:
done